### PR TITLE
Turn indicators

### PR DESCRIPTION
**Pull request by dcoollx**
this pull request adds a turn indicator to make it more clear who's turn it is

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,10 +7,11 @@
 	</head>
 	<body>
 		<center>
-            <input id="playersCount" type="number" min="2" value="2" max="8" style="width: 3em;"/>
+          <label style="color:white;">Number of players:</label>  <input id="playersCount" type="number" min="2" value="2" max="8" style="width: 3em;"/>
```
Comment (dcoollx) [inline index.html]: added a label so that the player number box's purpose is clear

**Comment by ketanhwr** [comment]
@dcoollx after testing properly it seems that the more than 2 players PR has infact a lot of bugs. I need to revert that change, which means you need to close this one too. I'm sorry :sweat:

**Comment by dcoollx** [comment]
np, but what exactly are the bugs that were introduced? ive played a couple games and see no apparent problems

**Comment by ketanhwr** [comment]
@dcoollx try playing with 8 players once. It's buggy. _pretty buggy_

**Pull request closed without merging**